### ISSUE TITLE
`childNode.data` is undefined

### ISSUE THREAD

**Issue by fiatjaf** [davidmerfield/Typeset#30]
I'm testing the client version from Chrome, using jQuery:

Sometimes [this line](https://github.com/davidmerfield/Typeset/blob/633f61f4ccc3ee5c39b675ada61784b044e3dcaf/src/eachTextNode.js#L38) doesn't work because `.data` is undefined.

```javascript
        childNode.data = doThis(childNode.data, childNode);
```

When it happens everything fails.

**Comment by lydell** [comment]
Fixed in #28.

**Comment by davidmerfield** [comment]
#28 was merged and this should be fixed. Closing for now.
